js/pages/configuration/configuration.js

```diff
@@ -61,7 +61,8 @@ define([
         }
       });
       
-		  this.canImport = ko.pureComputed(() => this.isAuthenticated() && authApi.isPermittedImportUsers());
+      this.canImport = ko.pureComputed(() => this.isAuthenticated() && authApi.isPermittedImportUsers());
+      sourceApi.initSourcesConfig();
```

Comment:
Data loading should take place in `onPageCreated` / `onRouterParamsChanged`

Reply:
updated.

Reply:
Also I added small change in SourceApi. I removed default value at source.version. This change prevent from blink during initialization.